src/BTVNanoCommissioning/utils/plot_utils.py

```diff
@@ -137,7 +137,7 @@ def plotratio(
     denom,
     ax=None,
     clear=True,
-    overflow="none",
+    flow=None,
```

Comment:
I think this has to be temporarily removed as well, until we have over and underflow functionality available

Reply:
same comments for the `autoranger` function